src/main/java/seedu/ddd/model/Model.java

```diff
@@ -59,6 +60,16 @@ public interface Model {
      */
     boolean hasContact(Contact contact);
 
+    /**
+     * Returns true if a person with the same identity as {@code person} exists in the address book.
```

Comment:
Should be `Returns true if a client with {@code id} exists in the address book.` instead.

Reply:
hasContact is used for adding contact, and to check if there are duplicates.